packages/react/src/components/InAppMessaging/MessageLayout/MessageLayout.tsx

```diff
@@ -118,7 +125,7 @@ export function MessageLayout({
               onClick={secondaryButton?.onAction}
               style={styles.secondaryButton}
             >
-              {secondaryButton?.title}
+              {secondaryButton.title}
             </Button>
           )}
           {hasPrimaryButton && (
```

Comment:
Curious about why `hasPrimaryButton` is needed, couldn't this be inferred from the `primaryButton` prop? Same question for `hasSecondaryButton` and `hasRenderableImage`. Seems like we could get into a scenario where the boolean value is true but the actual button/image props are missing?  Perhaps add a comment on `UseMessageProps` for future reference, to outline why these are needed.

Reply:
Believe this out of scope for this PR. While i agree that we could probably remove the button booleans,  believe the reason they were added was to increase the robustness of our code in the scenario that a customer passes the wrong type of value as `primaryButton` or `secondaryButton`. 

`hasRenderableImage` is not determined by just looking at the image object, and adding a comment abt `UseMessageProps` doesn't quite make sense to me in this file since the hook itself is not used in it, and could be missed if `useMessageProps` is modified in the future leaving an orphaned comment.